ximinez/rippled
# Testing merge / CI behavior

Testing merge / CI behavior

Initial state:
* test1 is at 724a301599, which is the commit before "0b812cdece Add RPC/WS ports to server_info" was merged.
* test2 uses test1 as a base, and has "refactor: rename ServerHandlerImp to ServerHandler" cherry-picked on to it.

Test steps:
1. Once the CI on this PR passes, bring test1 to 0e983528e1, which is the state of `develop` before "refactor: rename ServerHandlerImp to ServerHandler" was merged.
2. Force the CI build jobs to rerun.

Expected result:
1. The "checkout" action will checkout a merge commit of test1 and test2, which is implied by [Checkout pull request HEAD commit instead of merge commit](https://github.com/actions/checkout#checkout-pull-request-head-commit-instead-of-merge-commit)
2. The build will fail as the build failed in `develop` after "refactor" was merged.


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ximinez/rippled/2)
